packages/date-picker/src/panel/date.vue

```diff
@@ -214,7 +215,10 @@
     methods: {
       proxyTimePickerDataProperties() {
         const format = timeFormat => {this.$refs.timepicker.format = timeFormat;};
-        const value = value => {this.$refs.timepicker.value = value;};
+        const value = value => {
+          const defaultTime = this.defaultTime || [];
```

Comment:
As mentioned before, for non-range, defaultTime should be a single value, not an array